tools/assetpipelineeditor/main.qml

```diff
@@ -41,21 +41,6 @@ Kuesa.SceneEntity {
     readonly property Layer texturePreviewLayer: Layer {}
     readonly property Layer materialPreviewLayer: Layer {}
 
-    // let this point light wander around with the camera to create some shiny lighting
-    Entity {
-        id: pointLightEntity
-        property Camera followedCamera: frameGraph.camera
-        parent: followedCamera.parent
```

Comment:
I think the correct fix is actually parent: followedCamera no ?

Reply:
You are right. I forgot that QCamera is just a regular entity with a Transform-component.